src/status_im/constants.cljs

```diff
@@ -208,6 +208,8 @@
 (def ^:const path-ledger-live "m/44'/60'")
 (def ^:const path-keepkey "m/44'/60'")
 
+(def ^:const wallet-initial-derivation-path "m/44'/60'/0'/0/0")
```

Comment:
The derivation path by default for new accounts associated to new keypairs :eyes:

Reply:
🙏

Reply:
@ulisesmac we already have this defined:
```
(def ^:const path-wallet-root "m/44'/60'/0'/0")
(def ^:const path-default-wallet (str path-wallet-root "/0"))
```

You could rename it

Reply:
Thank you @OmarBasem ! I fixed it.